### PR TITLE
Correct version numbers and avoid ProductVersion in upstream

### PR DESCRIPTION
**Pull request by ekohl**
See individual commits for details.

### CODE REVIEW

File: guides/common/modules/proc_building-a-discovery-image.adoc [hunk 1]
```diff
@@ -77,8 +77,8 @@ repo --name=centos --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$ba
 repo --name=centos-updates --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=updates
 repo --name=epel7 --mirrorlist=https://mirrors.fedoraproject.org/metalink?repo=epel-7&arch=$basearch
 repo --name=centos-sclo-rh --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=sclo-rh
-repo --name=foreman-el7 --baseurl=http://yum.theforeman.org/{ProductVersion}/el7/$basearch/
-repo --name=foreman-plugins-el7 --baseurl=http://yum.theforeman.org/plugins/{ProductVersion}/el7/$basearch/
+repo --name=foreman-el7 --baseurl=http://yum.theforeman.org/{ProjectVersion}/el7/$basearch/
```
Comment (maximiliankolb) [inline guides/common/modules/proc_building-a-discovery-image.adoc:80]:
```suggestion
repo --name=foreman-el7 --baseurl=http://yum.theforeman.org/releases/{ProjectVersion}/el7/$basearch/
```

This URL only works for nightly, but not for any releases.

Comment (ekohl) [reply guides/common/modules/proc_building-a-discovery-image.adoc:80]: Good find. "fun" fact, it was already broken before.

**Review by maximiliankolb — APPROVED** [review]
LGTM!

**Comment by ekohl** [comment]
Anything needed here to move this forward?

**Comment by maximiliankolb** [comment]
I think you're good @ekohl  Are you hesitant to merge your own PR? If I knew, I would have immediately merged after my review. I don't know the process.

**Comment by ekohl** [comment]
Indeed I was. While I sometimes do that, it really depends. In this repository I generally prefer to not merge my own PRs.

**Pull request merged**